### ISSUE TITLE
Provide basic support for displaying NumericField for Currency

### ISSUE THREAD

**Issue by ns8482e** [OrchardCMS/OrchardCore#10490]
### Is your feature request related to a problem? Please describe.
For non-developer,  who's using Orchard Core CMS for their local web site - needs a way for `NumericFiled` to be displayed as Currency, without any programming knowledge

- Probably based on Site settings without Localization feature.

**Comment by Skrypt** [comment]
Are you needing absolutely the Site Settings without Localization feature?

**Comment by ns8482e** [comment]
Yes, to have Display and Editor option of `Currency`  that use above code.

However `CultureInfo` for currency to configured in Site Settings similar to time zone and not depend on `Localization` feature.  So that even with Localization and different culture for contents -  the currency stays same based to site setting.

**Comment by Skrypt** [comment]
Well never mind, I guess for a "basic" display that would make sense.

**Comment by ns8482e** [comment]
Yes - from site setting - because each tenant can set different setting independent of  OS `CultureInfo.CurrentUICulture`

**Comment by infofromca** [comment]
https://github.com/OrchardCMS/OrchardCore.Commerce/issues/678

**Comment by hishamco** [comment]
@infofromca luckly I replied to your issue there, because I already developed such field
